config/config.go

```diff
@@ -142,8 +142,10 @@ type Config struct {
 	NormalizeHost bool          `yaml:"normalize-host"`
 	HostPatch     net.HostPatch `yaml:"-"`
 
-	ValidateQuery bool      `yaml:"validate-query"`
-	RefusePayload multiFlag `yaml:"refuse-payload"`
+	ValidateQuery         bool      `yaml:"validate-query"`
+	ValidateQueryLog      bool      `yaml:"validate-query-log"`
+	ValidateQueryLogLevel string    `yaml:"validate-query-log-level"`
```

Comment:
I think we should drop the idea if having a log level here. @AlexanderYastrebov I think we can't do this because the logging.Logger should be used and we can't have 2 level settings here.

Reply:
Why `logging.Logger` and not `github.com/sirupsen/logrus`?

Reply:
Yeah but not as suggested before